core/state/statedb.go

```diff
@@ -1068,25 +1068,29 @@ func (s *StateDB) Commit(deleteEmptyObjects bool) (common.Hash, error) {
 //
 // Potential EIPs:
 // - Reset transient storage(1153)
-func (s *StateDB) Prepare(rules params.Rules, sender common.Address, dst *common.Address, precompiles []common.Address, list types.AccessList) {
+func (s *StateDB) Prepare(rules params.Rules, sender, coinbase common.Address, dst *common.Address, precompiles []common.Address, list types.AccessList) {
```

Comment:
Nitpick, can you also add the Shanghai Fork information here:

e.g. 

Shanghai fork:
- Add coinbase to access list(3651)